provider/azure/environ_test.go

```diff
@@ -295,7 +292,7 @@ func prepareForBootstrap(
 		makeResourceGroupNotFoundSender(".*/resourcegroups/juju-testmodel-model-deadbeef-.*"),
 		makeSender(".*/resourcegroups/juju-testmodel-.*", makeResourceGroupResult()),
 	}
-	env, err := environs.Open(stdcontext.TODO(), provider, environs.OpenParams{
+	env, err := environs.Open(stdcontext.Background(), provider, environs.OpenParams{
```

Comment:
Correct, we should really only use TODO in non-test code.

Reply:
Sadly we're inconsistent here. I was attempting to stick what whatever was used originally in each suite. I can do a sweep and make them all TODO()